### ISSUE TITLE
Green tint when using powervr filter with VGA cable mode

### ISSUE THREAD

**Issue by BParks21** [libretro/flycast#1168]
When using the powervr2 post processing filter with the VGA cable mode there is a strong green tint. The tint does not appear with the filter off. The filter on does not generate a green tint with the other two cable modes. Is this normal?

**Comment by leilei-** [comment]
Yeah, I tried to replicate the issues with real VGA output off a Dreamcast, so there's some ringing bleed, some green, but I didn't implement the annoying interference.  It could be split off to an option since it's not really a PowerVR2 process but a signal defect, IMO

**Comment by BParks21** [comment]
Now when it's enabled I just get a black screen.

**Comment by BParks21** [comment]
This appears to be fixed

**Comment by Daemit** [comment]
Still getting green tint on my end, in every output mode (VGA, TV RGB, TV Composite). The only way to get rid of it is to turn off the powerVR filter. BTW, what does this filter supposed to do?

**Comment by BParks21** [comment]
Is your core up to date?

**Comment by Daemit** [comment]
As far as I can tell, yes. Updated it through retroarch updater. Core version: 7d64554

The green tint is most noticeable on black screens or dark scenes with lots of black.

**Comment by BParks21** [comment]
I can't reproduce it what games are you testing?

**Comment by Daemit** [comment]
Alone in The Dark The New Nightmare, Resident Evil: Code Veronica X, Under Defeat. I can make screenshots, if it helps.

**Comment by BParks21** [comment]
Yeah let me see some screenshots. Also list your settings for the core. I tried Alone in The Dark, don't see anything.
